Fix FT.CREATE example and add example for document scoring (#1077)
RedisLabs/redislabs-docs: content/embeds/tryout-redisearch-2.md

### Creating indexes

Let's create a new index called "`database_idx`".
When you define the index, you must pass in the structure of the data you are adding to the index.
In this example, we have four fields: `title` (`TEXT`), `body` (`TEXT`), `url` (`TEXT`) and `visits` (`NUMERIC`), with the title field having a higher weight than the others (`5.0`).

1. Connect to Redis (replace 12543 with the port number your RediSearch is running on).

    ```sh
    $ redis-cli -p 12543
    127.0.0.1:12543>
    ```

2. Create the schema:

    ```sh
    127.0.0.1:12543> FT.CREATE database_idx PREFIX 1 "doc:" SCORE_FIELD "doc_weight" SCHEMA title TEXT body TEXT url TEXT visits NUMERIC
    ```

    This command indexes all of the hashes with the prefix "`doc:`" as well as all the future ones that will be created.

>     By default, all hashes get a score of 1.0, but if we need to be able to assign document-specific score to documents, we can do that too. We only need to specify in the schema the name of the hash element in which we will define the document score which will override the default of 1.0. This is done by using the `SCORE_FIELD` directive.

Reviewer: ```suggestion
    By default, all hashes get a score of 1.0, but if we need to assign a document-specific score to documents, we can do that too.
    We only need to specify in the schema the name of the hash element in which we will define the document score that overrides the default of 1.0. This is done by using the `SCORE_FIELD` directive.
```

Author: Sorry @bmansheim , I had committed some other changes without seeing your request. Applied now.

### Testing the index

Now add some data to this index. Here we add a hash with the key "`doc:1`" and the fields:

- title: "Redis Labs"
- body: "Primary and caching"
- url: <https://redislabs.com/primary-caching>
- visits: 108

```sh
127.0.0.1:12543> HSET doc:1 title "Redis Labs" body "Primary and caching" url "<https://redislabs.com/primary-caching>" visits 108
OK
```

> To add a document specific score, causing the document to appear higher or lower in results, set a value for the `doc_weight` field, which we specified as the `SCORE_FIELD` in the schema definition:

Reviewer: ```suggestion
To add a document specific score that causes the document to appear higher or lower in results, set a value for the `doc_weight` field.
We specified the `SCORE_FIELD` in the schema definition to hold the document weight value.
```
